server/planning/events/events_schema.py

```diff
@@ -311,4 +315,15 @@
         },
     },
     "extra": metadata_schema["extra"],
+    "translations": {
+        "type": "list",
+        "mapping": {
+            "type": "nested",
+            "properties": {
+                "field": not_analyzed,
+                "language": not_analyzed,
+                "value": not_analyzed,
```

Comment:
I think the value in general should be analyzed if it's for data like headline/description